### PR TITLE
Added HiddenUsersList

### PR DESCRIPTION
**Pull request by WardsParadox**
Adds the ability to specify the HiddenUsersList.

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.loginwindow.plist [hunk 1]
```diff
@@ -770,6 +771,35 @@ Availability: Available in macOS 10.13 and later.</string>
 			<string>Show input menu in login window</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
+		</dict>
+				<dict>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.loginwindow.plist]: Can you please fix the `<dict>` alignment here?

**Comment by apizz** [comment]
@WardsParadox can you please also bump the [pfm_version](https://github.com/erikberglund/ProfileManifests/blob/bc3adc397db6f280da404357186703dd551cc84b/Manifests/ManifestsApple/com.apple.loginwindow.plist#L814)?

File: Manifests/ManifestsApple/com.apple.loginwindow.plist (hunk 1 above)
Comment (WardsParadox) [reply Manifests/ManifestsApple/com.apple.loginwindow.plist]: Fixed in the latest commit.

**Comment by WardsParadox** [comment]
@apizz See changes in commits. 
Updated pfm_version to 5
Updated pfm_last_modified to current zulu time date
Fixed evil tabs (dict alignment)

**Comment by apizz** [comment]
@WardsParadox thank you!

**Pull request merged**